REVIEW.md

```diff
@@ -0,0 +1,192 @@
+
+All reviewers are expected to follow these guidelines and labels in their review.
+If a reviewer forgets, they may be reminded.
+If a reviewer refuses, they will be contacted about a violation of [Spyder’s Code of Conduct](https://github.com/spyder-ide/spyder/blob/master/CODE_OF_CONDUCT.md) under `Giving and gracefully accepting constructive feedback`.
+As with all aspects of the Code of Conduct, this applies to everyone involved in the discussion (including members of the Spyder organization on GitHub).
+
+Other notes:
+- Unless the reviewer has explicit permission on the conversation for that PR, reviewers should not commit to the same branch as the PR being reviewed.
+- If more than roughly 20% of review comments fall in the `[optional]` section described below, then it is worth adding this feedback in an issue linked to the PR instead of on the PR directly.
+- Whenever practical, reviewers should add comments asking for a specific change as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).
+
+## Reviewing process
+
+### For code-focused review
+
+#### Required items
+
+The following are areas the community expects to be covered in the review of the code on each PR.
+Not all of the following individual questions will be relevant for every PR (for example, a bug fix may not need to answer the same questions as a proposal for a new feature), but these are the themes to be addressed.
+
+Review that addresses these questions on a PR will be prefixed with `[required]` so that the PR author(s) are clear that they are blockers.
+PR authors are responsible for addressing required reviews for the PR to be eligible for merging.
+
+- [ ] Motivation
+    - Do we want this?
+    - Would it benefit a meaningful number of users to have this accepted into Spyder?
+    - Does it help to solve bugs?
+    - Is it a necessary refactoring?
+
+- [ ] Tests
+    - Do the tests pass in the continuous integration build?
+      If appropriate, help the contributor understand why tests failed.
+    - Are regression tests needed?
+      If a reproducible bug is being fixed in this PR, one may be needed, and reviewers should help the contributor design and implement them.
+      In case the contributor is unable to add the test, the reviewer should do so in a follow-up PR.
+
+ - [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy?
+    - Should variable names be improved for clarity or consistency?
+    - Should comments or docstrings be added or removed?
+    - Will the cost of maintaining a new feature be worth its benefits?
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Does the user experience match existing patterns in Spyder?
+      If it does not, why was the choice made?
```

Comment:
```suggestion
```

Looks like this was duplicated here along with below, possibly due to the merge conflict?